src/eventbridge/send.ts

```diff
@@ -3,24 +3,24 @@ import { EventEntry } from './EventEntry';
 import { Entries } from './Entries';
 import { SendResponse } from './SendResponse';
 import logger from '../observability/logger';
+import { Differences } from '../utils/differences';
 import { TestActivity } from '../utils/testActivity';
 
 const eventbridge = new EventBridge();
-const sendEvents = async (testResults: TestActivity[]): Promise<SendResponse> => {
+const sendEvents = async (events: Array<Differences | TestActivity>, type: string): Promise<SendResponse> => {
   logger.info('sendEvents starting');
-  logger.info(`${testResults.length} ${testResults.length === 1 ? 'event' : 'events'} ready to send to eventbridge.`);
+  logger.info(`${events.length} ${events.length === 1 ? 'event' : 'events'} ready to send to eventbridge.`);
 
   const sendResponse: SendResponse = {
     SuccessCount: 0,
     FailCount: 0,
   };
 
-  for (let i = 0; i < testResults.length; i++) {
-    const result = testResults[i];
+  for (let i = 0; i < events.length; i++) {
```

Comment:
May as well change this let to const

Reply:
`i++` can't make it const

Reply:
try `for (const event of events)`

Reply:
I tried that but eslint doesn't like it, although I could disable it. I think sonarqube doesn't like the current implementation though, so I guess make eslint unhappy or sonarqube unhappy 😄

Reply:
applied the `for (const event of events)` suggestion, had to disable the linter for it though